deployments/crds/hephaestus.dominodatalab.com_imagebuilds.yaml

```diff
@@ -222,6 +222,12 @@ spec:
                   - type
                   type: object
                 type: array
+              labels:
+                additionalProperties:
+                  type: string
+                description: 'Map of string keys and values roughly corresponding
+                  to Kubernetes Labels More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels'
```

Comment:
Not sure this is quite accurate, the image labels here for the most part don't map to k8s labels. It's still valuable to fetch and store them though, as some do and others (i.e. workspace properties) may be useful for other reasons.

Reply:
Right, then I've got it mixed up with OCI image config labels. Good catch, updated the description.

Reply:
https://github.com/dominodatalab/hephaestus/pull/134/files#diff-46b29db23c74cff80af70cc51499e89f78828c132fa4bbf743a0a684fe003522R228-R229

> Map of string keys and values corresponding OCI image config labels. Labels contains arbitrary metadata for the container